### PR TITLE
Playlist: adjust Moviecell to be reused by Playlist object

### PR DESCRIPTION
**Pull request by carolanitz**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md)
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

File: SharedSources/VLCMediaLibraryManager.swift [hunk 1]
```diff
@@ -87,6 +87,7 @@ class VLCMediaLibraryManager: NSObject {
     private lazy var medialib: VLCMediaLibrary = {
         let medialibrary = VLCMediaLibrary()
         medialibrary.delegate = self
+        medialibrary.setVerbosity(.debug)
```
Comment (carolanitz) [inline SharedSources/VLCMediaLibraryManager.swift]: needs to be removed

File: Sources/MediaCategoryCells/MovieCollectionViewCell.swift [hunk 2]
```diff
@@ -59,6 +63,16 @@ class MovieCollectionViewCell: BaseCollectionViewCell {
         newLabel.isHidden = !movie.isNew()
     }
 
+    func update(playlist: VLCMLPlaylist) {
+        collectionOverlay.isHidden = false
+        numberOfTracks.text = String(playlist.media.count)
+        titleLabel.text = playlist.name
+        descriptionLabel.text = playlist.description()
+//        if playlist.isThumbnailGenerated() {
```
Comment (carolanitz) [inline Sources/MediaCategoryCells/MovieCollectionViewCell.swift]: that should be removed and a ticket

File: VLC.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -752,8 +752,6 @@
...
 		7DBBF18F183AB4300009A339 /* VLCNetworkListCell.xib */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = file.xib; name = VLCNetworkListCell.xib; path = Resources/VLCNetworkListCell.xib; sourceTree = SOURCE_ROOT; };
 		7DBBF191183AB4300009A339 /* VLCMovieViewController.xib */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = file.xib; name = VLCMovieViewController.xib; path = Resources/VLCMovieViewController.xib; sourceTree = SOURCE_ROOT; };
-		7DBC85611A50B8860098D388 /* LiveAuthDialog_iPad.xib */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = file.xib; name = LiveAuthDialog_iPad.xib; path = ImportedSources/OneDrive/src/LiveSDK/Library/Internal/LiveAuthDialog_iPad.xib; sourceTree = SOURCE_ROOT; };
-		7DBC85621A50B8860098D388 /* LiveAuthDialog_iPhone.xib */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = file.xib; name = LiveAuthDialog_iPhone.xib; path = ImportedSources/OneDrive/src/LiveSDK/Library/Internal/LiveAuthDialog_iPhone.xib; sourceTree = SOURCE_ROOT; };
```
Comment (ePirat) [inline VLC.xcodeproj/project.pbxproj]: Why are those removed?

**Comment by carolanitz** [comment]
@ePirat that is a script that automatically shows which strings are not used anymore and removes them but I kept a couple around and but with the new uI these old librarystrings are not needed anymore. The live stuff happened also automatically but I make sure to not include that

**Comment by carolanitz** [comment]
merged with de32cbdfaf49b81293886d0cd2cbb66d86e5b64b
